.openpublishing.redirection.json

```
@@ -240,6 +240,10 @@
             "source_path": "docs/core/tutorials/cli-console-app-tutorial-advanced.md",
             "redirect_url": "/dotnet/core/tutorials"
         },
+        {
+            "source_path": "docs/core/tutorials/create-custom-template.md",
+            "redirect_url": "/dotnet/core/tutorials/cli-templates-create-item-template"
```

My suggestion for file name so you don't have template in there twice:

```suggestion
            "redirect_url": "/dotnet/core/tutorials/create-item-template-cli"
```

---

I would like to try and group them together some how. Since a new folder is out of the question, do you have a suggestion on a file name format that keeps the related files similar to find in a directory listing?